Ghost fleeing

ndirig opened the pull request.

ndirig committed:
Ghosts flee

Ghosts run away from PacMan and do not get stuck on a wall.

0x326 commented:
Should merge #27 before this one

0x326 reviewed (changes requested):
Can you make the ghosts slow down when they are vulnerable?

ndirig committed:
slowing vulnerable ghosts

The speed of ghosts decreases from 2.3 to 1.5 when PacMan eats a Power Pellet.  Speed is restored when the effect wears off.

0x326 committed:
Lint code

0x326 committed:
Merge branch 'master' into ghost-fleeing

0x326 committed:
Change speed when state changes

0x326 committed:
Run away in any other direction

0x326 committed:
Merge branch 'master' into ghost-fleeing

0x326 commented:
Seems like the ghosts sometimes approach Pac-Man when they are in tunnels (while they are still vulnerable)
